type-c-service/src/wrapper.rs

```diff
@@ -1,10 +1,10 @@
 //! This module contains the `Controller` trait. Any types that implement this trait can be used with the `ControllerWrapper` struct
 //! which provides a bridge between various service messages and the actual controller functions.
 use core::array::from_fn;
-use core::cell::RefCell;
+use core::cell::{Cell, RefCell};
 
 use bitfield::BitMut;
-use embassy_futures::select::{select, select_array, Either};
+use embassy_futures::select::{Either, select, select_array};
```

Comment:
Huh? `rustfmt` going nuts here?

Reply:
Does rustfmt sort ASCII? If so, capital is before lowercase.